src/polyswarm/links.py

```diff
@@ -30,3 +30,18 @@ def view(ctx, sha256):
     api = ctx.obj['api']
     output = ctx.obj['output']
     output.tag_link(api.tag_link_get(sha256))
+
+
+@link.command('list', short_help='List all links with the given tags.')
+@click.option('-t', '--tag', type=click.STRING, multiple=True)
+@click.option('-f', '--family', type=click.STRING, multiple=True)
+@click.option('-g', '--or-tag', type=click.STRING, multiple=True)
+@click.option('-y', '--or-family', type=click.STRING, multiple=True)
```

Comment:
can you add a description text for the or tag, or family args so people know what they mean?